streampark-flink/streampark-flink-proxy/src/main/scala/org/apache/streampark/flink/proxy/FlinkShimsProxy.scala

```diff
@@ -57,7 +57,7 @@ object FlinkShimsProxy extends Logger {
    * Get shimsClassLoader to execute for scala API
    *
    * @param flinkVersion flinkVersion
-   * @param func execute function
+   * @param func         execute function
```

Comment:
I think this line is no need to be changed.

Reply:
> I think this line is no need to be changed.

+1

Reply:
This is the IDEA formatting tool and has been rollback